src/main/java/io/cdap/e2e/utils/GcpClient.java

```diff
@@ -30,7 +30,7 @@
  */
 public class GcpClient {
 
-  public int countBqQuery(String table) throws IOException, InterruptedException {
+  public static int countBqQuery(String table) throws IOException, InterruptedException {
```

Comment:
Why?

Reply:
sorry i didnt understand this.
Actually this is being called in CdfHelper now